tsconfig.json

```diff
@@ -16,7 +16,6 @@
         "modules/contracts/deployments"
       ],
       "@connext/nxtp-subgraph": ["modules/subgraph/generated"],
-      "@connext/nxtp-sdk": ["modules/sdk/src"],
```

Comment:
... have there been two of these this whole time?

Reply:
yeah

Reply:
lol